pkg/glob/glob.go

```diff
@@ -10,13 +10,18 @@ import (
 
 // TODO: Use native path separators instead of always using /.
 
+type PathInfo struct {
```

Comment:
Exported symbols must have godoc comments.

Reply:
FYI, The existing code violates this best practice. A lot. I installed `golint` a few weeks ago. Simply doing `golint pkg/eval` identifies 24 such instances :-)

Reply:
Yes, but I unapologetically hold new code to a higher standard :)